code_of_conduct.md

```diff
@@ -0,0 +1,13 @@
+# Code of Conduct
+
+Online or off, Number Zero is a harrassment-free environment for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion or technical skill level. We do not tolerate harassment of participants in any form.
```

Comment:
"off" reads weird to me, I'd write "offline".  But there are folks with more native-speaker skills than me here.